templates/20-System Analysis/system-actor.html.j2

```diff
@@ -35,7 +35,7 @@
 
 <h2>Context</h2>
 <p>The context of {{ object.name }} is summarized by the diagram below:</p>
-{{ object.context_diagram.render("svg", display_derived_interfaces=True) | safe }}
+{{ object.context_diagram.render(None, display_derived_interfaces=True) | safe }}
```

Comment:
As mentioned at other locations. This change breaks the rendering of the diagram and we need to talk about that.

Reply:
My fault. Forgot to remove the ` | safe` part.